guides/source/active_storage_overview.md

```diff
@@ -721,10 +722,14 @@ Analyzing Files
 
 Active Storage analyzes files once they've been uploaded by queuing a job in Active Job. Analyzed files will store additional information in the metadata hash, including `analyzed: true`. You can check whether a blob has been analyzed by calling [`analyzed?`][] on it.
 
-Image analysis provides `width` and `height` attributes. Video analysis provides these, as well as `duration`, `angle`, and `display_aspect_ratio`.
+Image analysis provides `width` and `height` attributes. 
+Video analysis provides these, as well as `duration`, `angle`, `display_aspect_ratio` and presence/absence of `video` and `audio` channels.
 Audio analysis provides `duration` and `bit_rate` attributes.
 
-Analysis requires the `mini_magick` gem. Audio/Video analysis also requires the [FFmpeg](https://www.ffmpeg.org/) library, which you must include separately.
+Image analysis requires the `mini_magick` gem and the ImageMagick library, or the `ruby-vips` gem and the libvips library. 
+Audio/Video analysis requires the [FFmpeg](https://www.ffmpeg.org/) library.
+
+All three libraries mentioned above must be installed separately, as they are not included in the gems or rails.
```

Comment:
Since FFmpeg isn't really a library, what about this arrangement:

```suggestion
Image analysis requires either the `mini_magick` gem and the ImageMagick library, or the `ruby-vips` gem and the libvips library. Note that the libraries are not included in the gems, and must be installed separately.

Audio/Video analysis requires [FFmpeg](https://www.ffmpeg.org/), which must also be installed separately.
```

Reply:
I'm always confused about what's a library and what's not 😅

Reply:
Well I guess ImageMagick isn't really a library either, though I often hear it referred to as such.  (A library being something like a `.so` file on Linux or a `.dll` file on Windows.)  I think what we have is fine though. :+1: